src/components/BodyList.js

```diff
@@ -1,4 +1,4 @@
-/* eslint react/prop-types: 0 */import React, { useEffect } from 'react';
```

Comment:
- [ ] The `BodyList` component is a stateful component that deals with the redux store. So basically it is a `container` component. I recommend you create a `container` folder inside the `src` directory and move this file to the `container` folder for a more logical document structure. :pray: